lib/rsk-utils-legacy.js

```diff
@@ -2,6 +2,8 @@ const expect = require('chai').expect;
 const btcClientProvider = require('./btc-client-provider');
 const { sequentialPromise, wait, interval } = require('./utils');
 const { getBridgeAbi, getLatestActiveForkName } = require('./precompiled-abi-forks-util');
+const waitForBlockAttemptTimeMillis = process.env.WAIT_FOR_BLOCK_ATTEMPT_TIME_MILLIS || 200
```

Comment:
For consistency with the surrounding lines.

```suggestion
const waitForBlockAttemptTimeMillis = process.env.WAIT_FOR_BLOCK_ATTEMPT_TIME_MILLIS || 200;
```